banner_warn.php

```diff
@@ -36,6 +36,8 @@ function init()
             $RCMAIL = rcmail::get_instance();
 
             // Get config
+            $host =  $RCMAIL->config->get('org_email_regex');
+            $this->org_mail_regex = rcube_utils::parse_host($host);
             $this->org_mail_regex = $RCMAIL->config->get('org_email_regex');
```

Comment:
I'm very confused. Doesn't the value of `$this->org_mail_regex` just get overwritten here?